sensors/ev3_uart_sensor_defs.c

```diff
@@ -46,7 +46,7 @@ const struct ev3_uart_sensor_info ev3_uart_sensor_defs[] = {
 			},
 			[1] = {
 				/**
-				 * @description: Ambient light<br />Red LEDs off
+				 * @description: Ambient light<br />Blue LEDs on
```

Comment:
Yes, I am sure. I did some research...

http://bricks.stackexchange.com/questions/4898/light-on-color-sensor-is-blue